lib/stdlib/src/erl_lint.erl

```diff
@@ -317,6 +317,9 @@ format_error({unsafe_var,V,{What,Where}}) ->
 format_error({exported_var,V,{What,Where}}) ->
     io_lib:format("variable ~w exported from ~w ~s",
                   [V,What,format_where(Where)]);
+format_error({match_underscore_var, V}) ->
+    io_lib:format("variable ~w is matched. Use '_' if you meant to ignore it",
```

Comment:
I suggest "variable ~w is already bound. If you meant to ignore this value, use a different underscore-prefixed name or just '_'."

Reply:
Thanks!